src/server/controllers/deleteRecGame.ts

```diff
@@ -0,0 +1,23 @@
+import RecordedGameModel from "@/db/mongo/model/RecordedGameModel";
+import getMongoClient from "@/db/mongo/mongo-client";
+
+export type deleteRecGameParams = {
+  gameGuid: string;
+};
+
+export default async function deleteRecGame(
+  params: deleteRecGameParams
+): Promise<void> {
+  const gameGuid = params;
+  await getMongoClient();
+  try {
+    const existingGame = await RecordedGameModel.findOne({ gameGuid });
```

Comment:
I don't think you need to select it first - just try to delete and throw if it doesn't work. Would prefer 1 db query instead of 2.

Reply:
So this will delete the game from our recorded games collection, but and additional call will need to be made to S3 to delete the file from storage. I'm ok with this as a fast follow up story if you don't want to add it right now. I probably need to edit the bucket to allow deleting...but we can figure that out. Maybe create an issue for this if you aren't going to cover it as part of this PR.

Reply:
Done.
Will do the s3 thing in some other PR